spanner/spansql/types.go

```diff
@@ -208,6 +208,37 @@ const (
 	CascadeOnDelete
 )
 
+// AlterDatabase represents an ALTER DATABASE statement.
+// https://cloud.google.com/spanner/docs/data-definition-language#alter-database
+type AlterDatabase struct {
+	Name       ID
+	Alteration DatabaseAlteration
+
+	Position Position // position of the "ALTER" token
+}
+
+func (ad *AlterDatabase) String() string { return fmt.Sprintf("%#v", ad) }
+func (*AlterDatabase) isDDLStmt()        {}
+func (ad *AlterDatabase) Pos() Position  { return ad.Position }
+func (ad *AlterDatabase) clearOffset()   { ad.Position.Offset = 0 }
+
+type DatabaseAlteration interface {
+	isDatabaseAlteration()
+	SQL() string
+}
+
+func (SetDatabaseOptions) isDatabaseAlteration() {}
```

Comment:
Can we move this after the definition of `SetDatabaseOptions`?

Reply:
Done